[2024-02-15T09:52:32.802+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: spotify_to_rds_dag.extract_data_task manual__2024-02-15T09:51:29.239771+00:00 [queued]>
[2024-02-15T09:52:32.803+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: spotify_to_rds_dag.extract_data_task manual__2024-02-15T09:51:29.239771+00:00 [queued]>
[2024-02-15T09:52:32.803+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2024-02-15T09:52:32.807+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_data_task> on 2024-02-15 09:51:29.239771+00:00
[2024-02-15T09:52:32.812+0000] {standard_task_runner.py:60} INFO - Started process 99636 to run task
[2024-02-15T09:52:32.815+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'spotify_to_rds_dag', 'extract_data_task', 'manual__2024-02-15T09:51:29.239771+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/spotify_to_rds_dag.py', '--cfg-path', '/var/folders/85/0h16y03s0t59r_hcf0nwnhnr0000gn/T/tmphvhdodro']
[2024-02-15T09:52:32.816+0000] {standard_task_runner.py:88} INFO - Job 34: Subtask extract_data_task
[2024-02-15T09:52:32.842+0000] {task_command.py:423} INFO - Running <TaskInstance: spotify_to_rds_dag.extract_data_task manual__2024-02-15T09:51:29.239771+00:00 [running]> on host rachels-mbp
[2024-02-15T09:52:32.857+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='spotify_to_rds_dag' AIRFLOW_CTX_TASK_ID='extract_data_task' AIRFLOW_CTX_EXECUTION_DATE='2024-02-15T09:51:29.239771+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-02-15T09:51:29.239771+00:00'
[2024-02-15T09:52:32.858+0000] {http.py:169} INFO - Calling HTTP method
[2024-02-15T09:52:32.860+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/operators/python.py", line 199, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/operators/python.py", line 216, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/Library/Mobile Documents/com~apple~CloudDocs/Documents/dev/SpotifyETL/airflow/dags/spotify_to_rds_dag.py", line 34, in call_flask_app
    ).execute(context=kwargs)
      ^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/providers/http/operators/http.py", line 166, in execute
    return self.execute_sync(context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/providers/http/operators/http.py", line 170, in execute_sync
    response = self.hook.run(self.endpoint, self.data, self.headers, self.extra_options)
               ^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/providers/http/operators/http.py", line 148, in hook
    conn = BaseHook.get_connection(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/hooks/base.py", line 82, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/rachel/.local/share/virtualenvs/SpotifyETL-kuoYQhpC/lib/python3.11/site-packages/airflow/models/connection.py", line 479, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `http://127.0.0.1:5000` isn't defined
[2024-02-15T09:52:32.864+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=spotify_to_rds_dag, task_id=extract_data_task, execution_date=20240215T095129, start_date=20240215T095232, end_date=20240215T095232
[2024-02-15T09:52:32.871+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 34 for task extract_data_task (The conn_id `http://127.0.0.1:5000` isn't defined; 99636)
[2024-02-15T09:52:32.917+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-02-15T09:52:32.924+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
